src/omega_edit/utils.ts

```diff
@@ -138,14 +144,29 @@ export async function viewportSubscribe(
 export async function startOmegaEditServer(
   ctx: vscode.ExtensionContext,
   rootPath: string,
-  omegaEditPackageVersion: string
+  omegaEditPackageVersion: string,
+  port: number
 ): Promise<[vscode.Terminal, boolean]> {
   const [scriptName, scriptPath] = await omegaEditServer.setupServer(
     rootPath,
     omegaEditPackageVersion,
     ctx.asAbsolutePath('./node_modules/omega-edit')
   )
 
-  let terminal = await runScript(scriptPath, scriptName)
+  process.env.OMEGA_EDIT_SERVER_PORT = port.toString()
+
+  let terminal = await runScript(
```

Comment:
`let` could be `const` here.  Would it be worth sending the UI an info/warn message about the server starting up successfully or not?